src/ui/ProcedureSpecEditor.as

```diff
@@ -184,13 +231,17 @@ public class ProcedureSpecEditor extends Sprite {
 			makeLabel('Add string input:', 14),
 			makeLabel('Add boolean input:', 14),
 			makeLabel('Add color input:', 14),
+			makeLabel('Add menu input:', 14),
+			makeLabel('Add number menu input:', 14),
```

Comment:
Funny meme idea: somehow making the menu and number menu inputs on the same row. Nice meme!